Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -209,8 +209,16 @@ - (void)refreshTitle {
     } else {
         self.titleButton.hidden = NO;
     }
-    NSString *title = [NSString stringWithFormat:@"%@ %@", [mediaGroup name], ArrowDown];
-    [self.titleButton setTitle:title forState:UIControlStateNormal];
+    NSString *localizedOptionHint = NSLocalizedString(@"Tap here to change", "Tip for tapping media picker title to change the group.");
+    NSString *albumName = [NSString stringWithFormat:@"%@\n", [mediaGroup name]];
+    UIFont *titleFont = self.titleButton.titleLabel.font;
+    NSMutableAttributedString *title = [[NSMutableAttributedString alloc] initWithString:albumName attributes:@{NSFontAttributeName: titleFont}];
+    NSString *callForAction = [NSString stringWithFormat:@"%@ %@",localizedOptionHint, ArrowDown];
+    [title appendAttributedString:[[NSAttributedString alloc] initWithString:callForAction attributes:@{NSFontAttributeName: [titleFont fontWithSize:floorf(titleFont.pointSize * 0.75)]}]];
+
+    [self.titleButton setAttributedTitle:title forState:UIControlStateNormal];
+    self.titleButton.titleLabel.lineBreakMode = NSLineBreakByWordWrapping;
```

Comment:
I think we want truncation not word wrapping, in the case of long album titles:

<img width="350" alt="screen shot 2017-02-28 at 13 48 32" src="https://cloud.githubusercontent.com/assets/4780/23407606/abbb3f20-fdbc-11e6-93a4-f7393c7020d6.png">